src/pages/docs/channels/options/deltas.mdx

```diff
@@ -31,6 +31,10 @@ As delta compression is specified by a subscriber, the publisher has no control
 
 There is no constraint on how many publishers or subscribers there are. If there are multiple publishers, then deltas can still be generated, and they will be determined based on the order of messages. Deltas are calculated strictly based on the message ordering in that channel, with the effectiveness being dependent on the level of similarity between successive payloads.
 
+<Aside data-type='see-evidence'>
+Subscribers on that channel anywhere in the world will receive those messages in the same order they were originally published. [Click here for evidence 🔬](https://ably.com/docs/platform/architecture/message-ordering)
```

Comment:
This comment refers to "that channel" and "those messages", but it's unclear what it's referring to based on the surrounding text.
If this is stating that all subscribers globally see messages in the same order, for a given channel, then that's not true. So let's revisit this claim and decide what we need to say.